### ISSUE TITLE
Special Urls and YASD

### ISSUE THREAD

**Issue by HyperionEx** [conceptualspace/yet-another-speed-dial#63]
Hi,

With YASD i tried to do some sort of config panel in the SpeedDial (with the browser special pages/settings shortcuts).

Having urls like this:

- chrome://flags
- chrome://settings/content/notifications
- vivaldi://flags
etc ...

Is suppose YASD checks the urls against http:// or https://. So you can't do it ... Could you change this or provide some solution ?

Thanks !

**Comment by conceptualspace** [comment]
yes you're exactly right, because speed dials are actually bookmarks, (most) browsers impose the limitation that they have to be web protocols. the plan is for YASD to support these eventually, but i need to code in some special handling for them

**Comment by d9k** [comment]
@conceptualspace, what is a progress on issue?

**Comment by WoosterUK** [comment]
Would I be correct in inferring that this applies also to the file:// protocol? I can add these into the bookmark list just fine, and they function as 'normal' if I select through a bookmark manager, but they don't show in YASD.
